.ahoy.yml

```diff
@@ -159,9 +159,12 @@ commands:
         ) || [ "${ALLOW_BDD_FAIL:-0}" -eq 1 ]
       elif [ "$BEHAVE_TAG" = "authenticated" ]; then
         # run any tests that don't have a specific tag
-        ahoy cli "behave -k ${*:-test/features} --tags=-unauthenticated --tags=-smoke --tags=-OpenData" \
+        ahoy cli "behave -k ${*:-test/features} --tags=-unauthenticated --tags=-smoke --tags=-OpenData --tags=-multi_plugin" \
```

Comment:
why remove multi plugins?

Reply:
They're run separately. The idea is to split the tests in a somewhat logical and even fashion, to speed up the build.